src/main/kotlin/no/liflig/properties/GriidPropertiesFetcher.kt

```diff
@@ -6,11 +6,25 @@ import kotlinx.serialization.json.JsonObject
 import kotlinx.serialization.json.JsonPrimitive
 import java.util.Properties
 
-object GriidPropertiesFetcher {
+object GriidPropertiesFetcher : AwsPropertiesFetcher {
+
     private val serializer = JsonElement.serializer()
     private val json = Json {}
 
-    @JvmStatic
```

Comment:
Viktig at denne ikke fjernes da vi har andre klienter av dette biblioteket som er Java-applikasjoner

Reply:
For å kunne kjøre unit-tester med mocks måtte jeg fjerne static annoteringen. Har derfor også gjort om dette til en `class`.
I praksis betyr det at der hvor man tidligere gjorde 

`GriidPropertiesFetcher.forPrefix("prefix)` 
nå må gjøre 

`GriidPropertiesFetcher().forPrefix("prefix)`

Levbart kompromiss? :)